docs/podman-container-uninstall.1.md

```diff
@@ -0,0 +1,72 @@
+% PODMAN(1) Podman Man Pages
+% Dan Walsh
+% January 2015
+# NAME
+podman-container-uninstall - Remove/Uninstall container/container image from system
+
+# SYNOPSIS
+**podman container uninstall**
+[**--display**]
+[**-f**][**--force**]
+[**-h**|**--help**]
+[**-n**][**--name**[=*NAME*]]
+[**--storage**]
```

Comment:
not seeing storage as an option in the code.

Reply:
fixed